### ISSUE TITLE
[Bug]: No Auth Button available

### ISSUE THREAD

**Issue by NetRat88** [XeroxDev/Loupedeck-plugin-NanoleafControl#4]
### Consent

- [X] I have searched the existing issues and verified that there is no open issue for the same subject.
- [X] I understand that the Loupedeck NanoleafControl plugin has NO affiliation with Nanoleaf

### Current Behavior

After insallting Nanoleaf Plugin (on Windows) to my new Loupedeck Live S there is no authentication Button available to connect to the existing nanoleaf aurora.
![Bug_nanoleaf_auth_button](https://github.com/XeroxDev/Loupedeck-plugin-NanoleafControl/assets/6528507/1fcdd076-d1de-48f5-ae60-42dbedeb82cc)

### Expected Behavior

Searching for the possibility to connect to my existing nanoleaf aurora lights

### Steps To Reproduce

1. Install Plugin
2. Go to NanoleafControl
3. try to connect to exisisting nanoleafs

### OS

Windows

### OS Version

11

### Plugin Version

latest 3.0.0

### Nanoleaf devices

Nanoleaf Canvas (aurora)

### How did you download the plugin?

Loupedeck Store

**Comment by XeroxDev** [comment]
Hey!

It seems like the plugin can't find any Nanoleaf devices in your network.

Are they plugged in, turned on, connected to your network and configured?

**Comment by NetRat88** [comment]
Yes, they are up and running for years. :-) Also integrated in my FHEM Home Automation. How can I troubleshoot the issue?

Best regards

**Comment by XeroxDev** [comment]
Is your computer and the devices in the same network? No vpn, no sub-nets (both aren't supported), etc?

**Comment by NetRat88** [comment]
Yes both in the same network without VPN. Same Subnet etc.
![FritzBox_Nanoleaf_WindowsPC](https://github.com/XeroxDev/Loupedeck-plugin-NanoleafControl/assets/6528507/6a3182c6-a2c9-4da9-a964-c15b116a7766)

Also the Webinterface of the nanoleaf is reachable:

![nanoleaf_webinterface](https://github.com/XeroxDev/Loupedeck-plugin-NanoleafControl/assets/6528507/54fc7de1-f99f-43a4-b1ab-888fe09ea17e)

**Comment by XeroxDev** [comment]
Mh.. You said you've them now since years? Don't know the versions right now, but did you tried to update them? Maybe you're using a old/unsupported version?

**Comment by NetRat88** [comment]
9.2.4 is the latest Firmware version of the Canvas and they are up to date:

https://helpdesk.nanoleaf.me/en-US/canvas-firmware-release-notes-15618

**Comment by XeroxDev** [comment]
I tried to reproduce it but couldn't. Can you send me your exact Nanoleaf device and model?

**Comment by NetRat88** [comment]
These are the Nanoleaf Canvas as you can find here: https://nanoleaf.me/de-DE/products/nanoleaf-canvas/starter-kit-17-panels/

Firmware and Hardware version you can find in the screenshot above. What kind information should I provide?
To set it up on the Loupedeck you just have to install the plugin and then switch to the Nanoleaf plugin like in my screenshot and you should find the Auth button there, right? There's nothing more I can do to setting it up, right?

I also tried to uninstall and reinstall it.
I could also test it with my macbook but it seems that your pluginis only compatible with windows.

Otherwise I have to find a way to integrate my FHEM home automation to the loupedeck. With this it should also be possible to control the nanoleaf but I still not found a way, how to integrate devices from FHEM to the loupedeck.

**Comment by NetRat88** [comment]
What kind of ports or network protocoll your plugin needs to show up the auth button? Maybe it is a problem with the local windows firewall?! I am only using windows for business. So maybe the firewall is blocking something? But I need to know how the plugin is working to check the firewall settings etc.

best regards and thanks for yur help

**Comment by XeroxDev** [comment]
Thanks. I've still one question (just to confirm something). Did you install the official Nanoleaf Control Center (the official software to control your devices from your computer)? And do they find the devices?

**Comment by NetRat88** [comment]
What Nanoleaf Control Center software for computer you mean? I only have the offical Nanoleaf iOS App to controll the device installed on my iPhone. No software on computer.

EDIT: Ok alright. I found the Nanoleaf Desktop App (https://nanoleaf.me/de-DE/integration/desktop-app/). Currently I am not using this software. It seems that you need to have a nanoleaf cloud account. Is it really neccessary to get it up and running for the loupedeck? It's a pitty, then I'll try to find a solution to somehow integrate the Nanoleafs via my FHEM. Maybe there are some macros for this.

**Comment by XeroxDev** [comment]
Due to shared libraries which I'm not allowed to install (but Nanoleaf is), this software should be installed. Also it helps with debugging issues (if the desktop app can't connect, the plugin can't connect as well)
